webapp/src/components/EditAccountSettingsModal.vue

```diff
@@ -0,0 +1,146 @@
+      <template v-slot:header> Account settings </template>
+
+      <template v-slot:body>
+        <div class="form-row">
+          <div class="form-group col-md-6">
+            <label for="account-name" class="col-form-label">Name:</label>
+            <input
+              v-model="user.display_name"
+              type="text"
+              class="form-control"
+              id="account-name"
+              required
+            />
+          </div>
+        </div>
+        <div class="form-row">
+          <div class="form-group col-md-6">
+            <label for="account-name" class="col-form-label">Contact email:</label>
+            <input
+              v-model="user.contact_email"
+              type="text"
+              class="form-control"
+              id="account-name"
+              placeholder="Please enter your email"
+            />
+          </div>
+        </div>
+        <div class="form-row">
+          <div class="form-group col-md-6">
+            <a
+              v-if="user.identities.some((identity) => identity.identity_type === 'github')"
+              type="button"
+              class="dropdown-item btn login btn-link btn-default"
+              :href="
+                'https://github.com/' +
+                user.identities.find((identity) => identity.identity_type === 'github').name
+              "
+            >
+              <font-awesome-icon :icon="['fab', 'github']" />
+              {{ user.identities.find((identity) => identity.identity_type === "github").name }}
+            </a>
+
+            <a
+              v-else
+              type="button"
+              class="dropdown-item btn login btn-link btn-default"
+              aria-label="Login via GitHub"
+              :href="this.apiUrl + '/login/github'"
+              ><font-awesome-icon :icon="['fab', 'github']" /> Login via GitHub</a
+            >
+          </div>
+        </div>
+        <div class="form-row">
+          <div class="form-group col-md-6">
+            <a
+              v-if="user.identities.some((identity) => identity.identity_type === 'orcid')"
+              type="button"
+              class="dropdown-item btn login btn-link btn-default"
+              :href="
+                'https://orcid.org/' +
+                user.identities.find((identity) => identity.identity_type === 'orcid').name
+              "
+            >
+              <font-awesome-icon :icon="['fab', 'github']" />
+              {{ user.identities.find((identity) => identity.identity_type === "orcid").name }}
+            </a>
+            <a
+              v-else
+              type="button"
+              class="disabled dropdown-item btn login btn-link btn-default"
+              aria-label="Login via ORCID"
+              :href="this.apiUrl + '/login/orcid'"
+              ><font-awesome-icon class="orcid-icon" :icon="['fab', 'orcid']" /> Login via ORCID</a
```

Comment:
Minor rewording suggestion (also I think ORCID is formatted ORCiD -- probably we are not consistent with this ourselves...)

```suggestion
              ><font-awesome-icon class="orcid-icon" :icon="['fab', 'orcid']" /> Connect your ORCiD</a
```